### PR TITLE
Add Review component

### PR DESCRIPTION
**Pull request by c0m1t**
Consists of a title, a select-box and a limited-text-area-input.

### CODE REVIEW

File: src/shared/forminator/__stories__/SelectInput.stories.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ storiesOf('Forminator|Select input', module)
   .add('simple', () => {
     return (
       <Forminator onSubmit={action('submit')}>
-        <FormControl style={{ minWidth: 200 }}>
+        <FormControl variant="outlined" style={{ minWidth: 200 }}>
```
Comment (sijav) [inline src/shared/forminator/__stories__/SelectInput.stories.tsx]: Not necessary

File: src/shared/review/Review.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+type Props = FCProps<OwnProps>;
+
+function Review({ title, prefix }: Props) {
+  const { labelWidth, labelRef } = useLabelWidth();
+
+  return (
+    <Card>
+      <CardHeader title={title}></CardHeader>
+      <CardContent>
+        <DictInputItem field={prefix + '__rating'}>
```
Comment (smmoosavi) [inline src/shared/review/Review.tsx]: + 'Rating'

File: src/shared/review/Review.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          <Box width={240}>
+            <FormControl variant="outlined" margin="dense" fullWidth>
+              <InputLabel ref={labelRef}>{i18n._('Select')}</InputLabel>
+              {/* TODO: get options from GraphQl */}
+              <SelectInput options={[]} labelWidth={labelWidth} />
+            </FormControl>
+          </Box>
+        </DictInputItem>
+        <DictInputItem field={prefix + '__description'}>
```
Comment (smmoosavi) [inline src/shared/review/Review.tsx]: + 'Comment'

File: src/shared/review/Review.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import { FCProps } from 'src/shared/types/FCProps';
+
+interface OwnProps {
+  title: string;
+  prefix: string;
+}
+
+type Props = FCProps<OwnProps>;
+
+function Review({ title, prefix }: Props) {
```
Comment (smmoosavi) [inline src/shared/review/Review.tsx]: rename to `CriterionItem`

File: src/shared/criterion-item/CriterionItem.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import { Container, Grid } from '@material-ui/core';
+import { action } from '@storybook/addon-actions';
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import CriterionItem from 'src/shared/criterion-item/CriterionItem';
```
Comment (smmoosavi) [inline src/shared/criterion-item/CriterionItem.stories.tsx]: relative

**Pull request merged**